src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -1358,10 +1507,8 @@ def setColormapRangeFromDataBounds(self, bounds):
 
         :param Union[List[float],None] bounds:
             (xmin, xmax, ymin, ymax) Rectangular region in data space
+            If None, the full view is used
```

Comment:
It's not the full view, it's the item's visible area.

Reply:
I will respel it, but it's the exact same meaning.

The "view" is the visible area, else you talk about "data"